decisions/wc_offmap_decisions.txt

```diff
@@ -58,92 +67,130 @@ offmap_decisions = {
 
 		# DO NOT FORGET TO CHANGE "caltulate_PREV_offmap_old_gods_currency_insanity_effect" IF YOU CHANGE THIS!
 		# THIS IS ONLY NEEDED FOR SHOWING OFFMAP CURRENCY AMOUNT THAT YOUR CHARACTER WILL RECEIVE ON OG SCREEN THIRD PARTY SELECTOR
+		
 		third_party_score = {
 			factor = 100
-			
+
 			# Priests
 			additive_modifier = {
 				value = 500
-				FROMFROM = { 
+				FROMFROM = {
 					show_scope_change = no
-					controls_religion = yes 
+					controls_religion = yes
 				}
 			}
 			additive_modifier = {
 				value = 100
-				FROMFROM = { 
+				FROMFROM = {
 					show_scope_change = no
 					OR = {
 						is_priest = yes
 						is_ascetic_trigger = yes
 					}
 				}
 			}
-			
+
 			# Rulers and tiers
 			additive_modifier = {
 				value = 50
-				FROMFROM = { 
+				FROMFROM = {
 					show_scope_change = no
-					is_ruler = yes
+					tier = BARON
 				}
 			}
```

Comment:
I would change it back. These modifiers are different from what you see in `wc_sacrifice_scripted_effects`. It's confusing.